AmountInput/README.md

```diff
@@ -0,0 +1,58 @@
+
+# AmountInput Interface
+
+**Status**: Proposed
+
+| Name        | ID                                                                |
+| :---------- | :---------------------------------------------------------------- |
+| AmountInput | a1d347099e29c1624c8890619daf207bde18e92df5220a54bcc6d858309ece84  |
+
+
+## Description
+
+Interface for amount input.
+
+## Functions
+
+`get` - prints prompt message to the user and returns amount of tokens entered by user.
+
+arguments:
+
+	answerId: uint32 - id of function callback.
+	prompt: bytes - utf-8 string to print to the user before input.
+	decimal: uint8 - number of decimals used by amount token, e.g. 9 - means that browser should multiply token amount to 1000000000 to get its debot represenation.
+	min: uint256 - minimal amount that can be entered by user.
+	max: uint256 - maximum amount that can be entered by user.
+
+returns:
+
+	{ value: uint256 }
```

Comment:
Need to clarify to developer that the value is multiplied by 10^decimal here as well

Reply:
Done